src/middlewared/middlewared/utils/time_utils.py

```diff
@@ -0,0 +1,6 @@
+from datetime import datetime, UTC
+
+
+def time_now(naive=True):
+    """Wrapper for `datetime.now()`. Exclude timezone if `naive=True`."""
+    return datetime.now(None if naive else UTC)
```

Comment:
`datetime.now(None)` and `datetime.utcnow()` are different

Reply:
You're right, I'll have to go with [@aiden3c 's fix](https://github.com/truenas/middleware/pull/14194#issuecomment-2284094947).